.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: php
 php:
-  - 5.2
```

Comment:
Instead of removing 5.2 tests, we should try to fix them?
Or is there no phpunit version available for 5.2 anywhere?